master/usage/configuration/node.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Configuring a node
+---
+
+Each Calico node has key routing configuration that may be specified in a number of
+different ways.  In particular, this includes the IPv4 and IPv6 addresses and 
+subnets used for routing, and the BGP AS Number (which generally only needs to 
+be changed if you are explicitly configuring BGP peerings with your Calico nodes).
+
+To configure the routing parameters on your node, you have the options of:
+-  Set appropriate flags on the `calicoctl node run` command
+-  Set appropriate environments for the `calico/node` container
+-  Configure the node resource using `calicoctl`
+
+> Note that for nodes that were configured prior to v2.1, the subnet information 
+> configured on those nodes will be set to /32 for IPv4 or /128 for IPv6.  If you are
+> planning on using the `cross-subnet IP-in-IP` feature then you must update your node
+> configuration to have the correct subnet information for each node.
+
+## Configuring the IP address and subnet
```

Comment:
I think it would be useful to be more explicit about what the IP is used for. You  hint that it's used for BGP, but we should mention that it is the address which other nodes will use to make a BGP connection with this node.